.github/workflows/commits.yml

```
@@ -0,0 +1,59 @@
+name: Commit message validation
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - run: npm install --save-dev @commitlint/{config-conventional,cli}
+
+      - name: Create commitlint config file
+        run: |
+          cat << EOF > commitlint.config.js
+          module.exports = {
+            extends: ['@commitlint/config-conventional'],
+            rules: {
+              'type-enum': [2, 'always', [
+                'feat', 'fix', 'docs', 'style', 'refactor', 'test', 'chore', 'build', 'ci',
+                'enh', 'enhance', 'tweak', 'imp', 'improve',
+              ]],
+              'subject-case': [2, 'always', 'sentence-case'],
+              'header-max-length': [2, 'always', 72],
```

```suggestion
              'header-max-length': [2, 'always', 100],
```

This would make it more forgiving to longer headers, like what you had the other day.